ci/pipeline.yml

```diff
@@ -10,7 +10,7 @@ steps:
     command: '{{python}} ci/buildkite.py build {{name}}'
     key: '{{name}}'
     agents:
-      PLATFORM: '{{name}}'
+      PLAIDML_TARGET: '{{name}}'
```

Comment:
This is an important abstraction because platform may encompass more than just a plaidml target. For instance, we want to be able to tag machines that are capable of running non-plaidml compilers (like tensorflow, or whatnot).

Reply:
WAI -- experimental code